app/jobs/regular/bulk_user_title_update.rb

```diff
@@ -2,6 +2,7 @@
 
 module Jobs
   class BulkUserTitleUpdate < ::Jobs::Base
+    DESTROY_ACTION = 'destroy'.freeze
```

Comment:
This new constant seems unused and should probably be removed.

Reply:
Also a note ... all those .freeze calls are not needed since `# frozen_string_literal: true` will fix